workbaskets/views/ui.py

```diff
@@ -371,7 +339,7 @@ def run_business_rules(self):
 
     def get_success_url(self):
         form_action = self.request.POST.get("form-action")
-        if form_action in ("publish-all", "remove-selected"):
+        if form_action in ("remove-selected"):
```

Comment:
I guess this could now just be an equality check.
```suggestion
        if form_action == "remove-selected":
```